src/otx/engine/engine.py

```diff
@@ -127,10 +128,10 @@ def __init__(
                 meta_info=self._datamodule.meta_info if self._datamodule is not None else None,
             )
         )
-        self.optimizer: OptimizerCallable = (
+        self.optimizer: OptimizerCallable | None = (
```

Comment:
This is typically used for `OTXLitModule` builds. It shouldn't be able to accept `None` at `OTXLitModule`, but it's worth checking. If all is well, it looks fine.

Reply:
`optimizer` and `scheduler` can be set in `configure_optimizer`, but I skipped it. So, no problem.
```python
    def configure_optimizers(self) -> None:  # type: ignore[override]
        """Skip configure_optimizers unused in zero-shot visual prompting."""
```
https://github.com/sungchul2/training_extensions/blob/bb0a2c54e6bd03b6d726ef25e344496be1b933fe/src/otx/core/model/module/visual_prompting.py#L268-L269